### PR TITLE
Add Plot buy/selling

### PR DESCRIPTION
**Pull request by DerDevHD**
## Introduction
Hi. Y'all waited for Plot selling and buying. And now its here!

### Relevant issues
#9 

### API changes
There are added 2 new Methods for the MyPlot class:
```php
MyPlot::sellPlot(Plot $plot, float $price)
MyPlot::buyPlot(Plot $plot, Player $player)
```

### Behavioral changes
Economy is now registered onLoad instead of onEnable

## Backwards compatibility
It's 100% backwards compatible

## Follow-up
* Translations Needed
* Add buy and sell forms

## Tests
Tested it successfully without any errors.

### CODE REVIEW

File: resources/eng.ini [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ; Popup when entering a plot
 popup=You entered plot {%0}
 popup.owner=Owned by {%0}
+popup.forsale=This plot is for sale by {%0}. Use /p buy to claim the plot for ${%0}.
```
Comment (DerDevHD) [inline resources/eng.ini:8]: Wrong placeholder

Comment (jasonwynn10) [reply resources/eng.ini:8]: no, thats correct

Comment (jasonwynn10) [reply resources/eng.ini:8]: oh you mean the `{%0}`. I see

**Comment by HimbeersaftLP** [comment]
Doesn't this allow you to buy more plots than the maximum plot limit per player?

**Pull request merged**